static/src/javascripts/projects/common/modules/experiments/tests/support-epic-circles.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import { makeABTest } from 'common/modules/commercial/contributions-utilities';
+import { getSync as getGeoLocation } from 'lib/geolocation';
+
+const targetUrl = () => {
```

Comment:
Is there any way to share this function between the two tests rather than copy/pasting?

Reply:
Fair point. I think I wasn't sure if it'd work for both or whether one was geolocated and the other was edition-drive. Tests on CODE suggest it'd be ok to share though.